provision/acc_provision/templates/provision-config.yaml

```diff
@@ -17,6 +17,10 @@ aci_config:
     # type: vmware              # Specify the VMM vendor (supported: vmware)
     # name: myvmware            # Specify the name of the VMM domain
 
+    # Loadbalancer IP used to create duplicate service file for
+    # router-default service in Openshift on ESX case
+    # installer_provisioned_lb_ip: 192.168.18.201
```

Comment:
Sorry, one more small nit, can we say this is a "required" field in that case? Also, can we say that this should correspond to the Loadbalancer IP used in the installer?

Reply:
Added.